AUTONOMOUS_PLAN.md

```
@@ -0,0 +1,132 @@
+  lint:js` clean, `npm run build` success.
+
+---
+
+## [MODERN_WP_API] — Core API adoption & graceful fallbacks
+
+### MOD-1 `[COMPLETED - verified]` — WP_HTML_Tag_Processor usage
+- **Verified:** `class-telemetry.php`, `class-used-css.php`, `class-cache.php` (CDN rewrite),
+  `class-image-optimisation.php` already gate on `class_exists( 'WP_HTML_Tag_Processor' )`
+  with regex fallbacks. No action needed beyond spot-checking new code.
+
+### MOD-2 `[COMPLETED]` — Modern enqueue `wp_script_add_data` / strategy support
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)